spec/features/users/register_spec.rb

```diff
@@ -93,11 +93,11 @@
 
     expect(current_path).to eq('/register')
     expect(page).to have_content('Email has already been taken')
-    expect(page).to have_content(name)
-    expect(page).to have_content(address)
-    expect(page).to have_content(city)
-    expect(page).to have_content(state)
-    expect(page).to have_content(zip)
-    expect(page).to_not have_content(email)
+    expect(page).to have_field('Name', with: name)
```

Comment:
The test was not able to see into the pre-populated form in the way it was written before.